### ISSUE TITLE
[med] Chrome + Jaws: Success message after uploading image is not announced.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8624]
## Action Performed:
1. Using JAWS + Chrome browser, open the URL staging.new.expensify.com/r/85021985  
2. Open settings, navigate to the profile button using Tab key and press Enter key.                                                      
3. Go to the edit profile picture using Tab key and press Enter key.   
4. Upload any picture and listen to the screen reader.

## Expected Result:
Success message after uploading image should be conveyed to the users as soon as it appear on the page, so that user can be aware of that information.

## Actual Result:
Success message after uploading image is not announced.

HTML
`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" width="20" height="20" fill="#C6C9CA"><path d="M16.9 8.8h-5.6V3.1c0-.4-.2-.6-.6-.6H9.4c-.4 0-.6.3-.6.6v5.6H3.1c-.3.1-.6.3-.6.7v1.2c0 .4.2.6.6.6h5.6v5.6c0 .4.2.6.6.6h1.2c.4 0 .6-.2.6-.6v-5.6h5.6c.4 0 .6-.2.6-.6V9.4c.2-.4-.1-.6-.4-.6z"></path></svg>`

**Other Occurrences:**
>Same issue repro on PRs #8751, #8743, #8809, #8804, #9044, #9055, #9094, #9212, #9284, #8928, #9180, #9284, #9343
>Same issue also repro on PRs #8689 , #9262 -Invite sent information is not announced.
>Same issue also repro on PRs #8493 -password changed information is not announced.
>Same issue also repro on PRs #8948, #8790, #8757, #8896, #9025, #9026, #9082, #9133, #9134, #9095, #9206, #9022, #8878, #9203, #9213 -Message sent information is not announced.
>Same issue also repro on PRs #8816, #8834, #8953, #8760, #8873, #8877, #9064 -Money Requested/sent information is not announced.
>Same issue also repro on PRs #8982, #8733, #8957 -New message received information is not announced.

## Workaround:
Yes

## Area issue was found in:
Chat box

## Failed WCAG checkpoints
4.1.3

## User impact:
If any status message or any task completion information is not conveyed to the users then user will not able to identify whether task is completed or not and they require manual effort to identify the changes occur on the page.

## Suggested resolution:
Make sure there is an announcement for Screen reader user once the image has been uploaded.

Refer to:
https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html

## Platform:

- Web
- iOS
- Android
- Mobile Web

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/163134910-bd31c821-420b-448f-ac3c-5bb6018e9ed7.mp4
